### PR TITLE
added a NonBinary Option to "PatientSex"

### PR DESCRIPTION
**Pull request by cornelia247**
Updated the Patient sex to include Nonbinary option

### CODE REVIEW

**Comment by jennydaman** [comment]
Continuing from https://github.com/FNNDSC/ChRIS_ultron_backEnd/pull/438#issuecomment-1274886168, do you want to remove the `choices` kwarg altogether?

**Comment by cornelia247** [comment]
No I feel like that would be very confusing. Adding an "other" choice should round it up nice with the three we have now. What do you think?

**Comment by jennydaman** [comment]
@cornelia247 huge apology from me! I should have just looked it up... According to the DICOM standard, "M", "F", and "O" are the only three valid values for "PatientSex." Would you revert 9bd8d3440adf050a0df1770cefbf8c0c1f470d24? After doing so, I'll be happy to merge it.

File: chris_backend/feeds/views.py [hunk 1]
```diff
@@ -347,6 +347,8 @@ def perform_update(self, serializer):
         """
         Overriden to update feed's owners if requested by a PUT request.
         """
+        if not isinstance(self.request.data, dict) or not getattr(self.request.data, '_mutable', 1):
```
Comment (jennydaman) [inline chris_backend/feeds/views.py:350]: Why did you merge ea06f75dcb3310cd7915d609a06a2fb2698235a0?

I think you should simply do a

```shell
git reset --hard 14f5281a46f79db29b0769b2279f36ea5e770f06
git push --force
```

**Review by jennydaman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**